sunpy/map/mapbase.py

```diff
@@ -428,36 +443,50 @@ def observatory(self):
 
     @property
     def xrange(self):
-        """Return the X range of the image from edge to edge."""
-        # TODO: This should be reading from the WCS object
-        xmin = self.center.x - self.dimensions[0] / 2. * self.scale.x
-        xmax = self.center.x + self.dimensions[0] / 2. * self.scale.x
-        return u.Quantity([xmin, xmax])
+        """
```

Comment:
I am not even sure we should have these properties at all, because in all but the simplest WCSes they are total nonsense.